docs/docs/cmd/spo/site/site-chrome-set.md

```diff
@@ -0,0 +1,58 @@
+# spo site chrome set
+
+Set the chrome header and footer for the specified site
+
+## Usage
+
+```sh
+m365 spo site chrome set [options]
+```
+
+## Options
+
+`-u, --url <url>`
+: URL of the site collection to which you want to change the chrome header/footer
+
+`--headerLayout [headerLayout]`
+: Specifies the header layout to set on the site. Options: `Standard|Compact|Minimal|Extended`. Default `Standard`.
```

Comment:
If we always use a default value, it makes it impossible to update other properties but leave this property as-is. I think we shouldn't be using a default value and expect that if no value was specified, the particular property shouldn't be updated. Applies to all other properties with a default value as well.

Reply:
It seems to be the API that expects a default value. I can try what happens if we leave it out.